crates/uv/tests/it/pip_install_scenarios.rs

```diff
@@ -4177,7 +4177,7 @@ fn no_sdist_no_wheels_with_matching_python() {
       ╰─▶ Because only package-a==1.0.0 is available and package-a==1.0.0 has no wheels with a matching Python implementation tag (e.g., `cp38`), we can conclude that all versions of package-a cannot be used.
           And because you require package-a, we can conclude that your requirements are unsatisfiable.
 
-          hint: Wheels are available for `package-a` (v1.0.0) with the following Python tag: `graalpy310`
+          hint: While solving for CPython 3.8, found wheels for `package-a` (v1.0.0) with the following Python tag: `graalpy310`
```

Comment:
I considered doing the same for platform, but I'm not sure what to show exactly... I could say "Linux", but it might be a glibc incompatibility? If I say macOS, do I include the version, or the architecture?

Reply:
Separately, I don't love this message ("While solving for..."). Would love help.

Reply:
Ah glad you don't love it either, that was my initial impression.

I'll help think of something.